pkg/testutil/leak.go

```diff
@@ -28,6 +28,8 @@ var LeakOptions = []goleak.Option{
 	goleak.IgnoreTopFunction("google.golang.org/grpc.(*addrConn).resetTransport"),
 	goleak.IgnoreTopFunction("google.golang.org/grpc.(*Server).handleRawConn"),
 	goleak.IgnoreTopFunction("go.etcd.io/etcd/pkg/logutil.(*MergeLogger).outputLoop"),
+	// FIXME: Go 1.16 changes call stack, I have no idea where this comes from.
```

Comment:
Can remove the comment. For most of the contents of the list, we don't know where they came from.

Reply:
fine